onnxruntime/core/graph/model.cc

```diff
@@ -447,7 +448,13 @@ Status Model::Load(int fd, ONNX_NAMESPACE::ModelProto& model_proto) {
   }
 
 #if GOOGLE_PROTOBUF_VERSION >= 3002000
-  FileInputStream input(fd);
+  size_t file_size = 0;
+  int block_size = -1;
```

Comment:
-1 is the default as per https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.io.zero_copy_stream_impl#FileInputStream.FileInputStream.details